style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+  border: 0px;
+}
+
+body {
+  background-color: #CCC;
+  font-family: Georgia, serif;
+  padding: 0px;
+  position: relative;
+  min-height: 100%;
```

Comment:
If you want this line to actually do anything (make the body the entire page height) you need to set the HTML's height  min-height to 100% as well.